alembic/versions/f492e8ed9d84_add_magnitude_and_exposure_time_columns_.py

```diff
@@ -0,0 +1,28 @@
+"""add magnitude and exposure time columns to the instrument table
+
+Revision ID: f492e8ed9d84
+Revises: f9611048498a
+Create Date: 2022-01-27 10:06:33.987883
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = 'f492e8ed9d84'
+down_revision = '82f5a89fbe59'
```

Comment:
will need to update this down revision hash given that there are new migrations.